pkgs/os-specific/linux/switcheroo-control/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, ninja, meson, fetchFromGitLab, systemd, libgudev, pkg-config, glib, python3Packages, gobject-introspection }:
+python3Packages.buildPythonApplication rec {
+  pname = "switcheroo-control";
+  version = "2.3";
+  
+  format = "other";
+  src = fetchFromGitLab {
+    hash = "sha256-1Pze2TJ9mggfcpiLFwJ7/9WhsdJx4G3GoA7+Z47shuc=";
+    domain = "gitlab.freedesktop.org";
+    owner = "hadess";
+    repo = pname;
+    rev = version;
+  };
+  
+  nativeBuildInputs = [
+    ninja meson pkg-config glib
```

Comment:
Could you add a comment what is glib needed for? I assume something like glib-mkenums?